app/views/layouts/application.html.erb

```diff
@@ -26,9 +26,13 @@
           </div>
           <div class="collapse navbar-collapse" id="myNavbar">
             <ul class="nav navbar-nav navbar-right">
-              <li><%= link_to "Sign Up", new_user_path, :id=> "links" %></li>
-              <li><%= link_to "Log In", login_path, :id=> "links" %></li>
-              <li><%= link_to "Log Out", logout_path, :id=> "links" %></li>
+              <%if current_user %>
```

Comment:
FYI there is also a `link_to_if`
http://apidock.com/rails/ActionView/Helpers/UrlHelper/link_to_if